src/main/groovy/liquibase/harness/util/TestUtils.groovy

```diff
@@ -12,33 +10,6 @@ import java.util.logging.Logger
 
 class TestUtils {
 
-    static SortedMap<String, String> resolveInputFilePaths(DatabaseUnderTest database, String basePath, String inputFormat) {
```

Comment:
This method was covered by TestUtilsTest.getInputFilesPaths test. Now this test is failing (but we don't see it in build)

Reply:
++ moved to file utils